src/database.js

```diff
@@ -28,7 +29,7 @@ export class Database {
       records.push(this.decorateRecord(collectionName, record));
     }
 
-    this.store[collectionName] = records;
+    this._pushToStore(collectionName, records);
```

Comment:
It's just a reminder that this method should only be used as database internally, it still can be used outside of it but it tells user that he probably shouldn't do this manually.